### PR TITLE
initial adjustments required

### PR DESCRIPTION
**Pull request by katietz**

### CODE REVIEW

**Comment by ELundby45** [comment]
@katietz Ready for review?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,11 +24,13 @@ requirements:
     - ninja
     - meson
     - pkg-config
-    - glib  # [build_platform != target_platform]
+    - patch           # [osx]
   host:
     - gobject-introspection
     - glib
     - zlib
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:31]: Does it not require `gio` too? See https://github.com/GNOME/gcab/blob/ad0baea50359c1978a9224ee60bf98d97bfb991f/meson.build#L97 and https://docs.gtk.org/gio/index.html.   I'm not familiar with meson, if it is not found dependency it won't generate an error?

Comment (katietz) [reply recipe/meta.yaml:31]: no, it is not required AFAICS

Comment (cbouss) [reply recipe/meta.yaml:31]: gio is included in glib

**Review by cbouss — APPROVED** [review]
LGTM

**Review by varlackc — COMMENTED** [review]
The following changes might be useful:

doc_source_url: https://github.com/GNOME/gcab/blob/master/README.md
license_url: https://github.com/GNOME/gcab/blob/master/COPYING
description: A GObject library to create cabinet files

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**